packages/flutter/test/material/user_accounts_drawer_header_test.dart

```diff
@@ -13,11 +13,13 @@ const Key avatarC = Key('C');
 const Key avatarD = Key('D');
 
 Future<void> pumpTestWidget(
-  WidgetTester tester, {
-  bool withName = true,
-  bool withEmail = true,
-  bool withOnDetailsPressedHandler = true,
-}) async {
+    WidgetTester tester, {
```

Comment:
Seems not aligned at here? (All arguments)

```suggestion
  WidgetTester tester, {
```

Reply:
IMO it is quite normal since 'tester' is not a named argument but a positioned one.